python/hail/matrixtable.py

```diff
@@ -2984,7 +2984,6 @@ def from_rows_table(cls, table: Table) -> 'MatrixTable':
         -------
         :class:`.MatrixTable`
         """
-        hail.methods.misc.require_key(table, 'from_rows_table')
```

Comment:
Can MatrixTables handle having an empty row key everywhere right now? The rvd partitioner stuff should work out fine, but do we make assumptions about the length of the row key elsewhere?

Reply:
Good point. `require_key` only checks if the key is `None`, but maybe we should make it also check if the key is `[]`. I would expect this to be temporary, and that everything (or at least most things) should work fine with empty key.